Source/common/SNTConfigurator.m

```diff
@@ -296,6 +298,9 @@ - (instancetype)initWithSyncStateFile:(NSString *)syncStateFilePath
     _syncStateFilePath = syncStateFilePath;
     _syncStateAccessAuthorizerBlock = syncStateAccessAuthorizer;
 
+    _defaults = [NSUserDefaults standardUserDefaults];
```

Comment:
Might be worth adding a comment for our future selves that this is to keep KVO on changes but we're using CF* for reading.